### PR TITLE
Focus Automate on Magic

### PR DESCRIPTION
**Pull request by wodsuz**
move strategy list to main page 
move login page to /login 
force magic theme for all users (disable theme matching logic)
on login screen, change 'Don't have an account? Sign up' to 'Don't have an account? Limited time First 87 $MAGIC✨ users Free sign up'

### CODE REVIEW

File: src/hooks/useTheme.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useAuth } from './useAuth';
 export const useTheme = () => {
   const { user } = useAuth();
   const { theme, setTheme } = useContext(ThemeContext);
-
+  console.log('theme: ' + JSON.stringify(theme));
```
Comment (mzolkiewski) [inline src/hooks/useTheme.ts:9]: remove

File: src/contexts/ThemeContext.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const ThemeContext = createContext<IThemeContext>({
 export const ThemeProvider: React.FC<IProps> = ({ children }: IProps) => {
   const [theme, setNewTheme] = useState(initialTheme);
   const setTheme = useCallback((name: string) => {
-    const newTheme = _setTheme(name, initialTheme.name);
+    const newTheme = _setTheme('magic', initialTheme.name);
```
Comment (mzolkiewski) [inline src/contexts/ThemeContext.tsx:30]: This is not DRY (Don't Repeat Yourself). What you did in this file is you changed the theme in a couple different places. Why not just make it so there's only 1 place? Just change line `35` so you always return the default theme (`theme: defaultThemeName`). The rest of the changes does not matter. Please also add a comment above that line to say something like `// force the default theme` so if we come back to this some time later then it all makes sense (it's one of those rare cases when a comment can actually be useful)

File: src/components/Auth/Auth.tsx [hunk 3]
```diff
@@ -132,6 +134,9 @@ const Container = styled.div`
     padding-left: 40px;
     padding-right: 40px;
   }
+  .SignupLoginText {
```
Comment (mzolkiewski) [inline src/components/Auth/Auth.tsx:137]: let's name classes in `camelCase`

**Review by mzolkiewski — CHANGES_REQUESTED** [review]

File: src/components/Scheduleds/Scheduleds.tsx [hunk 4]
```diff
@@ -17,6 +17,7 @@ import { SentinelAPI } from '../../api/SentinelAPI';
 import { TokenAPI } from '../../api/TokenAPI';
 import { bigNumberToNumber, normalizeBigNumber, numberToBn, shortAddress } from '../../utils';
 import { IScheduledForUser } from '../../types';
+import BlockExplorer from '../Transactions/BlockExplorer';
```
Comment (mzolkiewski) [inline src/components/Scheduleds/Scheduleds.tsx:20]: export and import via `/Transactions/index.ts`

File: src/constants.ts [hunk 5]
```diff
@@ -23,3 +23,17 @@ export enum ConfirmationTime {
   threeDays = '3d',
   fiveDays = '5d',
 }
+export enum NetworkScanUrl {
```
Comment (mzolkiewski) [inline src/constants.ts:26]: BlockExplorerUrl

File: src/constants.ts [hunk 6]
```diff
@@ -23,3 +23,17 @@ export enum ConfirmationTime {
...
+  None = '',
+  Arbitrum = 'https://arbiscan.io/',
+  Ethereum = 'https://etherscan.io/',
+  Ropsten = 'https://ropsten.etherscan.io/',
+  ArbitrumRinkeby = 'https://testnet.arbiscan.io/',
+}
+export enum BlockchainExplorer {
```
Comment (mzolkiewski) [inline src/constants.ts:33]: BlockExplorerName

File: src/hooks/useTheme.ts [hunk 7]
```diff
@@ -6,7 +6,6 @@ import { useAuth } from './useAuth';
 export const useTheme = () => {
   const { user } = useAuth();
   const { theme, setTheme } = useContext(ThemeContext);
-
```
Comment (mzolkiewski) [inline src/hooks/useTheme.ts:9]: Why remove this?

File: src/components/Transactions/BlockExplorer.tsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import { ExportOutlined } from '@ant-design/icons';
+
+import { shortAddress } from '../../utils';
+import { ChainId, NetworkScanUrl, BlockchainExplorer } from '../../constants';
+interface IProps {
+  address: string;
+  chars?: number;
```
Comment (mzolkiewski) [inline src/components/Transactions/BlockExplorer.tsx:7]: No need for this parameter, let's just force a consistent number of chars - 4

File: src/components/Transactions/BlockExplorer.tsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+  chainId: number;
+  isCheckingTx: boolean;
+  displayedText?: any;
+}
+
+function BlockExplorer({ address, chars, chainId, isCheckingTx, displayedText }: IProps) {
```
Comment (mzolkiewski) [inline src/components/Transactions/BlockExplorer.tsx:13]: BlockExplorerLink

File: src/components/Transactions/BlockExplorer.tsx [hunk 10]
```diff
@@ -0,0 +1,37 @@
+import { ExportOutlined } from '@ant-design/icons';
+
+import { shortAddress } from '../../utils';
+import { ChainId, NetworkScanUrl, BlockchainExplorer } from '../../constants';
+interface IProps {
+  address: string;
+  chars?: number;
+  chainId: number;
+  isCheckingTx: boolean;
+  displayedText?: any;
```
Comment (mzolkiewski) [inline src/components/Transactions/BlockExplorer.tsx:10]: remove this, and instead use `children`, that way you can do stuff like:
```jsx
<BlockExplorerLink ...>
   <ExportOutlined />
   some text
</BlockExplorerLink>
```
And this also allows you to remove that super ugly `if (displayedText === 'ScanMenuItem') {`

If no `children` passed, then you can default to `shortAddress`

File: src/components/Transactions/BlockExplorer.tsx [hunk 11]
```diff
@@ -0,0 +1,37 @@
+import { ExportOutlined } from '@ant-design/icons';
+
+import { shortAddress } from '../../utils';
+import { ChainId, NetworkScanUrl, BlockchainExplorer } from '../../constants';
+interface IProps {
+  address: string;
+  chars?: number;
+  chainId: number;
+  isCheckingTx: boolean;
```
Comment (mzolkiewski) [inline src/components/Transactions/BlockExplorer.tsx:9]: Remove this (why did you call it `checking`? are we checking anything here?). Instead, you should replace this and `address` with something like:
```
hash: string;
type: 'address' | 'tx';
```

**Review by mzolkiewski — CHANGES_REQUESTED** [review]

File: src/hooks/useTheme.ts (hunk 7 above)
Comment (wodsuz) [reply src/hooks/useTheme.ts:9]: i think i did this accidentally, apologies.

**Comment by mzolkiewski** [comment]
Resolve merge conflicts and we can merge

**Pull request merged**